File: 2019-06-25/is_shuffle_of/att/template.cpp
// template di soluzione per il problema is_shuffle_of
#include <cassert>
#include <fstream>

using namespace std;

const int MAXN = 200000;
const int MAXM = 200000;

int main() {
  ifstream fin("input.txt");
  assert(fin);
  int m,n,with_cert;
  fin >> m >> n >> with_cert;
  ofstream fout("output.txt");
  fout << 0 << endl;
  fin.close();
  fout.close();
  return 0;
}

File: 2019-06-25/is_shuffle_of/att/template.py
# -*- coding: utf-8 -*-
# template di soluzione per il problema is_shuffle_of

# template di soluzione che, pur con sintassi di python3, possa girare anche entro environment pensati per supportare python2

from __future__ import print_function
import sys
if sys.version_info < (3, 0):
    input = raw_input # in python2, l'equivalente di input è raw_input

fin = open("input.txt", "r")
m, n, with_cert = map(int,fin.readline().split())

fout = open("output.txt", "w")
fout.write("0")

fin.close()
fout.close() 
